Shared/AppConstants.swift

```diff
@@ -4,6 +4,9 @@
 
 import UIKit
 
+// Ecosia: moving the last version profile key
+public let LatestAppVersionProfileKey = "latestAppVersion"
```

Comment:
Sorry I couldn't find it, but for what is this used?

Reply:
That’s what I did in the first place yeah 🤣. Although I thought that the Coordinators shouldn’t have any other “behaviour” assigned to them implicitly.

Reply:
Found it in `AppLaunchUtil.swift` 👍